src/triton_fil/triton_utils.h

```diff
@@ -66,6 +66,21 @@ ModelStateType* get_model_state(TRITONBACKEND_Model& model) {
   return model_state;
 }
 
+inline TRITONSERVER_MemoryType get_native_memory_for_instance(
```

Comment:
This is a little heavy for an inline function in a header file. Let's separate the implementation to avoid the compilation cost.